test/quantization/fx/test_quantize_fx.py

```diff
@@ -5006,6 +5006,48 @@ def forward(self, x):
         quantized = convert_fx(prepared, is_reference=True)
 
 
+    @skipIfNoFBGEMM
+    def test_ave_pool_with_custom_cfg(self):
+        """ A test that checks correct patterns are produced for
+        all supported general value ops like aten::avg_pool2d \
+        without actually checking for execution of these ops
+        """
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.avg_pool2d = torch.nn.AvgPool2d(3)
+
+
+            def forward(self, x):
+                x = self.avg_pool2d(x)
+                return x
+
+        # This model is not executable since we just put all ops
```

Comment:
should we make these comments reflect the code?  Seems like these are copy-pasted, but not sure if they reflect this callsite.

Reply:
this comment is copied I think, we can remove it